requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django>=2.1.3,<2.2.0
```

Comment:
Better use fixed versions in your "production environment" to have a determined version deployed. Ranges are better suited for libraries to be as compatible as possible.

Reply:
I think I'll fix this at 2.2.24 in the first instance as 2.2 is the LTS versions which will support until 04/2022 and hopefully there will be less chance of breaking changes compares to 3.x. Ideally I'd have gone for the same version used in the TK repos